contracts/rosca.sol

```diff
@@ -1,72 +1,266 @@
 pragma solidity ^0.4.4;
 
+/*
+*  Possible missing functions
+* - integrity check? to perform check and make sure certain values are available and correct
+*
+*/
 contract rosca {
-    address WETRUST = 0;  // constant
-    
-    event LogParticipantRegistered(string uniqName);
-    event LogContributionMade(string uniqName);
-    event LogBidPlaced(string uniqName);
-    event LogRoundFundsRelease(string winnerUniqName, uint amountInWei);
-    
-    
-    enum Period { Weekly, Monthly }
-    
+    address constant WETRUST = 0x0;
+
+    enum Period { Weekly, Monthly } // are we only allwing 2 options?
+
+    event LogParticipantRegistered(address user);
+    event LogContributionMade(address user, uint amount);
+    event LogNewLowestBid(uint bid,address winnerAddress);
+    event LogRoundFundsRelease(address winnerUniqName, uint amountInWei);
```

Comment:
sorry, it's an abbreviation i use a lot, i should have checked... :)

s/something/different means: change "something" to "different".
(it's coming from vi commands: http://vim.wikia.com/wiki/Search_and_replace